pkg/pillar/cmd/zedagent/attesttask.go

```diff
@@ -28,6 +28,9 @@ const (
 	attestWdName            = agentName + "attest"
 	eventLogBinarySizeLimit = 2048  // 2KB
 	maxQuotePayloadSize     = 63488 // 62KB
+
+	savedEscrowStateFile = "escrow-state" // existence indicates that we sent storage keys
+	savedEscrowStateSent = "sent"         // the content of the savedEscrowStateFile if we sent keys
```

Comment:
You don't check the content, thus just an empty file is sufficient to carry the state.

Reply:
You are right, removed. The idea was to have several states inside the file, but seems it is not required now.